js/translator.js

```diff
@@ -0,0 +1,20 @@
+var languageChoice = ['en', 'vi', 'it', 'uk']
+
+var rand = (Math.floor(Math.random())*languageChoice.length);
+var rValue = languageChoice[rand];
+console.log(rValue)
+
+    if (languageChoice === "en") {
+        console.log('Hello World')
+    }
+    else if (languageChoice === "vi") {
```

Comment:
languageChoice is the array [] with the options in it, these comparison will always fail